src/lost_messages/pub_lost_msg.py

```diff
@@ -0,0 +1,41 @@
+import pika
+import json
+
+EXCHANGE = "ack-ex"
+ROUTING_KEY = "ack-rk"
+HOST = "localhost"
+QUEUE = "ack-q"
+TTL = 1000
```

Comment:
```suggestion
TTL = 1000 # in milliseconds
```
It took me a bit to realize that this are not seconds but milliseconds. So maybe we can document it here.